srcpkgs/glibc/patches/glibc-upstream-13.patch

```diff
@@ -50,3 +50,4 @@ index cf0213ece5..85239cedbf 100644
        new_argv[0] = (char *) _PATH_BSHELL;
        new_argv[1] = (char *) args->file;
        if (argc > 1)
+
```

Comment:
accidental extra line ?

Reply:
Added by the script that pulls the patches. I don't think it will hurt as the other patches also have a newline at the end of the file. This was the only one that hadn't.

Reply:
As long as it works nothing against